commons/com.b2international.index.api/src/com/b2international/index/revision/Revision.java

```diff
@@ -76,6 +90,11 @@ public long getStorageKey() {
 	public static final String COMMIT_TIMESTAMP = "commitTimestamp";
 	public static final String SEGMENT_ID = "segmentId";
 	public static final String REPLACED_INS = "replacedIns";
+	
+	/**
+	 * Revision fields that should not be part of any hash value.
+	 */
+	public static final Set<String> REV_FIELDS = ImmutableSet.of(BRANCH_PATH, COMMIT_TIMESTAMP, SEGMENT_ID, REPLACED_INS);
```

Comment:
They are not part of the `_source` value, and the _hash is computed from that by removing the metafields (except the `storageKey`) from the revision document.